src/app/components/Item/ItemHoldings.jsx

```diff
@@ -127,7 +127,7 @@ class ItemHoldings extends React.Component {
         />
       );
 
-      items = this.state.chunkedItems[this.state.page + 1];
+      items = this.state.chunkedItems[this.state.page - 1];
```

Comment:
True, but this case will only happen if `items` has more than 20 items, in which case `chuckedItems` will not be empty.